relationships/candidates/AWSAUTOSCALINGGROUP.yml

```diff
@@ -0,0 +1,14 @@
+category: AWSAUTOSCALINGGROUP
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSAUTOSCALINGGROUP
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: ["aws.instanceId"]
+          field: instanceId
```

Comment:
Is this tag specific for autoscaling groups? Or does every AWS element have it?

Reply:
Yes, Thats specific for AWS Auto Scaling Groups.